v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
I don't know how we word this usually, so feel free to ignore this, but it might be more correct to phrase this as "The `ARRAY` data type *is* a one-dimensional, ..."